### DISCUSSION TITLE
Aliases for GPIO devices to use in commands/rules such as Power-fan instead of Power1

### DISCUSSION THREAD

**Discussion by prabbit237** [arendst/Tasmota#13536] [Ideas]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._

I have some multi-outlet devices. So I have Relay1, Relay2 and Relay 3. If I plug in three devices, such as a cable modem, a router and a printer, it would be much easier if I can set an alias for each relay.

There's the "BLEAlias mac=alias mac=alias" command that solves the same issue for multiple Bluetooth devices and a GPIOAlias could do the same here. So I could do something like:

````
GPIOAlias 2=Modem 14=Router
````

And then later, without trying to recall if the modem is on outlet 1 or 2, etc, I could use commands like:

````
backlog Power-Modem 0;delay 600;Power-Modem 1
````
The controls can be named in the WebUI but if I'm doing all this from rules/console/remote MQTT, it'd be a pain to load up the WebUI just to remind myself which is whine.

**Comment by mrkeuz** [comment]
+1, also use case for Home-Assistant, for display sensors and assign persistent ID. I.e. in `ESPHome` you can assign name to any exported entity. Very useful feature.

**Reply by blakadder** [reply]
That is what FriendlyName is for

**Reply by mrkeuz** [reply]
Sorry, maybe I didn't found it in documentation. Is it possible to set FriendlyName for individual components within one tasmota device?
